packages/bpk-docs/src/components/DocsPageBuilder/NeoDocsPageBuilder.js

```diff
@@ -194,7 +195,8 @@ const NeoDocsPageBuilder = props => {
   ];
 
   const menu = [...props.components, ...(props.customSections || [])];
-  const showPageHead = props.tokenMap || props.blurb || menu.length > 0;
+  const showPageHead =
+    props.showPageHead && (props.tokenMap || props.blurb || menu.length > 0);
```

Comment:
I made a prop to override if the page head (title and menu) is shown, which we're making use of on this page.